packages/accounts-controller/src/AccountsController.ts

```diff
@@ -429,6 +435,29 @@ export class AccountsController extends BaseController<
     });
   }
 
+  /**
+   * Updates the metadata of the account with the given ID.
+   *
+   * @param accountId - The ID of the account to set the name for.
+   * @param metadata - The new metadata for the account.
+   */
+  setAccountMetadata(
```

Comment:
I'm on the fence here, given the method description and code, it could be called `updateAccountMetadata` instead.

Reply:
You're right @danroc , thanks! Updated!